src/App.tsx

```diff
@@ -25,9 +25,9 @@ export const App = () => {
     <>
       {isReady && (
         <div className={styles.app}>
-          <div
-            className={`${styles.backdrop} ${isMobileSideBarOpen ? styles.isOpen : ''}`}
-            onClick={() => setIsMobileSideBarOpen(false)}
+          <MobileBackdrop
+            styles={`${styles.backdrop} ${isMobileSideBarOpen ? styles.isOpen : ''}`}
+            onBackdropClick={() => setIsMobileSideBarOpen(false)}
```

Comment:
Componentized this to show its purpose. At first I though this empty div could have been left over from a past refactor, but now its purpose is immediately clear.